src/transformers/convert_slow_tokenizer.py

```diff
@@ -1140,6 +1140,17 @@ def post_processor(self):
         )
 
 
+class SiglipConverter(SpmConverter):
+    def post_processor(self):
```

Comment:
is there nothing else? Would assume the normalizer needs some NFC or special handling of the punctions? A Regex replace or whatever to canonicalize the text?

Reply:
?